crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -18,6 +18,7 @@ export namespace kinds {
     ArrayExpression = "ArrayExpression",
     ArrayValuesList = "ArrayValuesList",
     AsciiStringLiteralsList = "AsciiStringLiteralsList",
+    AssemblyFlagsDeclaration = "AssemblyFlagsDeclaration",
```

Comment:
looks like it is not covered by any tests. maybe we can add one as well?

Reply:
Added one in 4c1e674a0567238b8320125a96c237dd01d46854